pkg/controller/direct/bigqueryreservation/assignment_controller.go

```diff
@@ -179,19 +180,19 @@ func (a *AssignmentAdapter) Create(ctx context.Context, createOp *directbase.Cre
 
 	log.V(2).Info("successfully created Assignment", "name", a.id.String())
 
-	status := &krm.BigQueryReservationAssignmentStatus{}
+	status := &krmv1alpha1.BigQueryReservationAssignmentStatus{}
 	status.ObservedState = BigqueryReservationAssignmentObservedState_FromProto(mapCtx, created)
 	if mapCtx.Err() != nil {
 		return mapCtx.Err()
 	}
 
-	// update the externalRef in the KRM resoruce
```

Comment:
Nit: This change is probably wrong.  Although it's only a comment....

Reply:
🦅 👁️